### PR TITLE
Add lighthouse workflow

### PR DESCRIPTION
**Pull request by jvntf**
Add generic lighthouse workflow

### CODE REVIEW

**Comment by necoline** [comment]
maybe it's time to upgrade node to 12?
https://github.com/GoogleChrome/lighthouse/issues/11854

**Comment by AliceR** [comment]
I wonder if it would be easier if applied onto #729 ? At least no hassle with gulp

**Comment by necoline** [comment]
great job Jeevan! It's great to see these tests results. These reports aren't that surprising. I think the next step will be documenting how to develop with lighthouse reports. I don't the goal is make these general health checks rather than development blockers

**Comment by jvntf** [comment]
yes i agree, perhaps this app should be added to the repo to show reports? https://github.com/apps/lighthouse-ci

Lighthouse action should ideally not show as a failed test.

**Comment by necoline** [comment]
Let's add it in! I think we should be able to customize how it runs in the config file. I took an initial stab at removing our failing tests. The idea conveyed in the docs is that these assertions can acts as a todo list for improving the site, once addressed the failures highlight regressions in the code

**Comment by jvntf** [comment]
@necoline @AliceR do you know if we can safely remove the `npm install lhci` and `npm rebuild node-sass` from the workflow? im not sure if the former needs to be installed every time. the latter was needed after changing the node version, hopefully not that its been run it doesnt need to be included?

**Comment by necoline** [comment]
I'm not sure about installing `lhci` but it seems reasonable to remove the `node-sass rebuild`

File: .env.development [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Environment variables used by the app
```
Comment (AliceR) [inline .env.development:1]: ups, these were not meant to be committed

Comment (necoline) [reply .env.development:1]: 😬

**Pull request closed without merging**